### PR TITLE
show first page in modal as eaily as possible

### PR DESCRIPTION
**Pull request by Priestch**

### CODE REVIEW

File: extras/wasm/template/index.html [hunk 1]
```diff
@@ -181,7 +286,8 @@
                 console.log('Loading document...');
 
                 let o = {
-                    wasm: FPDF.LoadMemDocument(wasmBuffer, fileSize, "")
+                    wasm: FPDF.LoadMemDocument(wasmBuffer, fileSize, ""),
```
Comment (paulocoutinhox) [inline extras/wasm/template/index.html:289]: I think that this code is wrong, no? The var `wasmBuffer` is a param of what?

Comment (Priestch) [reply extras/wasm/template/index.html:289]: @paulocoutinhox It do the same as
```javascript
let o = {
    wasm: FPDF.LoadMemDocument(wasmBuffer, fileSize, ""),
    wasmBuffer: wasmBuffer,
};

Comment (paulocoutinhox) [reply extras/wasm/template/index.html:289]: Nice. Can you put as `wasmBuffer: wasmBuffer,` only for readability? Thanks.

Comment (Priestch) [reply extras/wasm/template/index.html:289]: @paulocoutinhox I just pushed a new commit, fixed the feedback.

**Comment by paulocoutinhox** [comment]
thanks @Priestch

**Pull request merged**